CHANGELOG.md

```diff
@@ -3,6 +3,28 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 2021-12-06
```

Comment:
Versioning and change logs not needed in the PR (melos automatically versions)

Please remove this change